controllers/Products.php

```diff
@@ -54,7 +54,10 @@ public function __construct()
             // existence and doesn't inherit the parent product's data if it exists.
             session()->flash('mall.variants.disable-inheritance');
 
-            if (str_contains(\Request::header('X-OCTOBER-REQUEST-HANDLER',''), 'PriceTable')) {
+            $requestFromOctoberCMS = str_contains(\Request::header('X-OCTOBER-REQUEST-HANDLER',''), 'PriceTable');
+            $requestFromWinterCMS  = str_contains(\Request::header('X-WINTER-REQUEST-HANDLER',''), 'PriceTable');
+            
+            if ($requestFromOctoberCMS || $requestFromWinterCMS) {
                 $this->preparePriceTable();
             }
```

Comment:
You could potentially use this block of code to remain compatible with both Winter and October

```php
if (str_contains($this->getAjaxHandler() ?? '', 'PriceTable')) {
    $this->preparePriceTable();
}
```

Reply:
Great suggestion, thanks @bennothommo!